frappe/core/doctype/data_import/data_import.py

```diff
@@ -110,4 +110,4 @@ def export_fixture(doctype, app):
 	if not os.path.exists(frappe.get_app_path(app, "fixtures")):
 		os.mkdir(frappe.get_app_path(app, "fixtures"))
 
-	export_json(doctype, frappe.get_app_path(app, "fixtures", frappe.scrub(doctype) + ".json"))
+	export_json(doctype, frappe.get_app_path(app, "fixtures", frappe.scrub(doctype) + ".json"), order_by="name asc")
```

Comment:
why this default?

Reply:
So that it won't affect any other functions calling export_json without order_by, expecting it to export result sorted by creation